src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -716,6 +716,41 @@ public function get_test_theme_version() {
 		return $result;
 	}
 
+	/**
+	 * Tests if the revision is on/off.
+	 *
+	 * @since 6.7.0
+	 *
+	 * @return array The revision results.
```

Comment:
```suggestion
	 * @return array The test results.
```

To be in sync with other functions.